src/nSPV/nspv_fullnode.cpp

```diff
@@ -34,6 +34,7 @@ std::map<NSPV_ERROR_CODE, std::string> nspvErrors = {
     { NSPV_ERROR_INVALID_RESPONSE, "could not create response message" },
     { NSPV_ERROR_BROADCAST, "could not broadcast transaction" },
     { NSPV_ERROR_REMOTE_RPC, "could not execute rpc" },
+    { NSPV_ERROR_REMOTE_RPC, "could not execute rpc" },
```

Comment:
I do not believe this is what you intended. Same index, same string, maps to same entry. Also, the last element ending with a comma seems strange.

Reply:
thank you fixed